docs/NetCore_GettingStarted.md

```diff
@@ -39,7 +39,7 @@
 
 1. Navigate to the root of the Roslyn-analyzers repo and run these commands:
     - `cd roslyn-analyzers`
-    - `set RUNTIMEPACKAGEVERSION=3.0.0`
+    - Set RUNTIMEPACKAGEVERSION variable with the version which ever the runtime/roslyn-analyzers repo is using: `set RUNTIMEPACKAGEVERSION=5.0.0`
```

Comment:
Could you point to a file or something where we could get the info?

Reply:
Sure, added a link